source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.js

```diff
@@ -311,16 +314,17 @@ export default class Step1ConfigurationDialog extends Component<Props> {
 
     const closeButton = <DialogCloseButton onClose={onClose} />;
 
-    const minDelegationFunds = MIN_DELEGATION_FUNDS;
+    const minRewardFunds = MIN_REWARDS_FUNDS;
 
-    const dropdownError = errorMessage && (
-      <p className={styles.errorMessage}>
-        <FormattedHTMLMessage
-          {...errorMessage}
-          values={{ minDelegationFunds }}
-        />
-      </p>
-    );
+    if (error) {
+      error = <p className={styles.error}>intl.formatMessage(error)</p>;
```

Comment:
Please, change the UI error into a different variable, so `error` prop is a constant. It's not good to change a variable type (from a `LocalizableError` into a React element).

Reply:
Fixed